webapp/src/blocks.tsx

```diff
@@ -1956,9 +1956,13 @@ function clearTemporaryAssetBlockData(workspace: Blockly.Workspace) {
     forEachImageField(workspace, field => field.clearTemporaryAssetData());
 }
 
-function setHighlightWarning(block: Blockly.BlockSvg, enabled: boolean) {
+async function setHighlightWarning(block: Blockly.BlockSvg, enabled: boolean) {
     (block.pathObject as PathObject).setHasError(enabled);
     block.setHighlighted(enabled);
```

Comment:
Does setting the highlight and then resizing make it so there's a snapping effect for the highlight? Or is it fast enough that there's no visual difference?

Reply:
there's no visual difference